lightning/restore/tidb.go

```diff
@@ -154,7 +156,15 @@ func InitSchema(ctx context.Context, g glue.Glue, database string, tablesSchema
 
 	task := logger.Begin(zap.InfoLevel, "create tables")
 	var sqlCreateStmts []string
-	for tbl, sqlCreateTable := range tablesSchema {
+	tables := make([]string, 0, len(tablesSchema))
+	for tbl := range tablesSchema {
+		tables = append(tables, tbl)
+	}
+	// sort tables for failpoint test
```

Comment:
I think a better solution is to use regexp to match the execute result in failpoint test